Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
+                                     using textStyle: TextStyle) -> NSAttributedString {
+    guard let (parsedString, elements) = try? MarkdownParser.parse(markdownText) else {
+        return NSAttributedString(string: markdownText, textStyle: textStyle)
+    }
+    
+    let attributedString = NSMutableAttributedString(string: textStyle.textTransform.applied(to: parsedString))
+    attributedString.addAttributes(textStyle.attributes,
+                                   range: NSRange(location: 0, length: parsedString.characters.count))
+    
+    elements.forEach { (element) in
+        var font: UIFont? = nil
+        var strikethroughStyle: NSUnderlineStyle? = nil
+        
+        switch element {
+        case .em(_):
+            font = textStyle.emFont
+        case .strong(_):
+            font = textStyle.strongFont
+        case .strikethrough(_):
+            strikethroughStyle = textStyle.strikethroughStyle
+        }
+        
+        if let font = font {
```

Comment:
Would probably be helpful to have functions dealing with font, strikethrough style and color, just for better readability